src/epcpm/cli/sunspectostaticmodbus.py

```diff
@@ -30,229 +28,63 @@ def map_type_uuid(input, sunspec_types, staticmodbus_types):
     raise ValueError("Error: failure to map type")
 
 
-def parse_sunspec_data_point(
-    data_point,
-    sunspec_types,
-    staticmodbus_types,
-    table_ref_uuid_map=None,
-    scale_factor_uuid_map=None,
+def create_function_data(
+    input_sunspec_data, scale_factor_uuid_map, sunspec_types, staticmodbus_types
 ):
-    print(f"data_point UUID: {data_point['uuid']}")
-    data_point["_type"] = replace_data_point_with_function_data(data_point["_type"])
-    data_point["type_uuid"] = map_type_uuid(
-        data_point, sunspec_types, staticmodbus_types
+    function_data = dict()
+    # TODO: This will likely be be split up into a separate method when bitfields and tables are implemented.
+    if input_sunspec_data["class_name"] == "DataPoint":
+        function_data["_type"] = "function_data"
+    elif input_sunspec_data["class_name"] == "DataPointBitfield":
+        function_data["_type"] = "function_data_bitfield"
+    else:
+        function_data[
+            "_type"
+        ] = f"Unsupported class_name '{input_sunspec_data['class_name']}' in create_function_data."
```

Comment:
I didn't really think this one through. I figured there was a low likelihood that the code would go down this route. The idea is that grid-tied would not build with the `_type` set this way and the message would give an indication of where the problem originated. In hindsight, the code might as well just error out immediately, because the problem would need to be fixed there anyways.

Reply:
Addressed in 56ddfd94f353c9f023fd8e6e56e3a0bd83b07283. Code raises an exception immediately for this situation since it is more important to know immediately.